tethys_cli/manage_commands.py

```diff
@@ -35,10 +38,13 @@ def add_manage_parser(subparsers):
     manage_parser.add_argument('-l', '--link', required=False, action='store_true',
                                help='Only used with collectstatic command. Link static directory to STATIC_ROOT '
                                     'instead of copying it. Not recommended.')
+    manage_parser.add_argument('--django-help', action='store_true',
```

Comment:
Is this necessary? Wouldn't `--help` just get passed through to the django command if they provided it? e.g.: `tethys manage shell --help` instead of `tethys manage shell --django-help`.

Reply:
This is what I was trying too explain yesterday. "--help" calls the tethys help so I had to create the "--django-help" to avoid conflict and still be able to pull the help from django using tethys manage.